Source/WebKit/UIProcess/API/Cocoa/WKWebView.mm

```diff
@@ -4153,7 +4153,7 @@ - (void)_createIconDataFromImageData:(NSData *)imageData withLengths:(NSArray<NS
             targetLengths.append(lengthValue);
     }
 
-    auto buffer = WebCore::SharedBuffer::create(imageData);
+    Ref buffer = WebCore::SharedBuffer::create(imageData);
```

Comment:
Not related to this patch, I would be tempted to not touch this file in this patch.